### ISSUE TITLE
javadoc not working with openjdk 11

### ISSUE THREAD

**Issue by hohwille** [devonfw/devon4j#108]
We fail to build our releases with openjdk11 due to this issue:
https://bugs.openjdk.java.net/browse/JDK-8212233

See also #16 

So to build releases of devon4j we either need to stick to jdk 8 or go to 12 (not yet testet) or use oraclejdk 11.

**Comment by hohwille** [comment]
It is working with 11.0.4 (that we use from AdoptJDK). However, some additional config in maven is still required before we can close the release.

**Comment by hohwille** [comment]
This issue caused the release to fail:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.0.0:jar (attach-javadocs) on project devon4j-test: Execution attach-javadocs of goal org.apache.maven.plugins:maven-javadoc-plugin:3.0.0:jar failed.: NullPointerException -> [Help 1]
```
Seems as if first the javadoc plugin needs to be updated.

**Comment by hohwille** [comment]
We need `3.1.0` as already documented here:
https://github.com/devonfw/devon4j/blob/develop/documentation/guide-jdk.asciidoc#maven-plugins

**Comment by hohwille** [comment]
Further, we need to set source to 8:
https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8212233
As already done for devon-ide:
https://github.com/devonfw/devon-ide/blob/85d599c8bf8824904a6cc1ee6279bc7bad40c24d/pom.xml#L149

**Comment by hohwille** [comment]
So the required changes to our POM are done with PR #140.
We further switched to AdoptOpenJDK in `devon-ide` so we get jdk `11.0.4` that contains the necessary fixes for javadoc tool.
